src/main/java/net/lenni0451/spm/commands/subs/Download_Sub.java

```diff
@@ -18,13 +18,19 @@ public class Download_Sub implements ISubCommandMultithreaded {
 
     @Override
     public boolean execute(CommandSender sender, String[] args) {
-        if (args.length != 3) return false;
+        if (args.length < 3) return false;
         if (!args[0].equalsIgnoreCase("direct") && !args[0].equalsIgnoreCase("spigot")) return false;
         if (!sender.hasPermission("pluginmanager.commands.download." + args[0].toLowerCase())) {
             Logger.sendPermissionMessage(sender);
             return true;
         }
 
+        String filename = args[2];
```

Comment:
The ``filename`` is unused. You would have to replace all ``args[2]`` usages below

Reply:
Resolved